### ISSUE TITLE
1.20 support 

### ISSUE THREAD

**Issue by ghost** [maksim789456/DynamicSurroundingsFabric#36]
It'd be nice :)

**Comment by maksim789456** [comment]
I am currently working on these

**Comment by vico93** [comment]
> I am currently working on these

Great to hear that!

**Comment by vico93** [comment]
@CreativeMD i think you would like to know about this

**Comment by Tapacywka** [comment]
> I am currently working on these

I understand that if development of 1.20.1 began after 1.19.2, then the version on 1.19.4 can not be expected?

**Comment by maksim789456** [comment]
> > I am currently working on these
> 
> I understand that if development of 1.20.1 began after 1.19.2, then the version on 1.19.4 can not be expected?

Version for 1.19.3 and 1.19.4 present in the repository, but I don't want to release it to the public yet because of one serious bug

**Comment by Tapacywka** [comment]
> Version for 1.19.3 and 1.19.4 present in the repository, but I don't want to release it to the public yet because of one serious bug

May I ask what kind of bug are we talking about?

**Comment by Thefatsparrow** [comment]
am glad theres gunna be 1.20 support!! have you got an estimate of the release date? like in a couple weeks or a month

thanks!

**Comment by MotMomet** [comment]
please make it happen 1.20 version

**Comment by Rebel459** [comment]
_I'm moderately excited for the port lol_

**Comment by SilverET456** [comment]
Favorite mod hands down. I eagerly look forward to this port!

**Comment by SnowPowerCore** [comment]
Would be very happy to install it onto 1.20.2

**Comment by Thanos1716** [comment]
This is my favorite sound mod by far, Resounding is good but also needs updating, and has some performance issues, and compatibility with Ambient Sounds iirc, and Sound Physics Remastered is alright but lacking 1.20.2 support. Neither sound as great as DSF in my opinion. Hoping to see this amazing mod make a come back soon!

**Comment by Rebel459** [comment]
Yeas, DSF is the best. OreCrunch has now ported to 1.20.4, so here’s hoping that between both these great devs we see a 1.20.1 port (I’ve already asked OreCrunch, they’ve confirmed that they’ll look into doing it in the future provided it isn’t too much work and they get the time). Just thought I’d update anyone here too

**Comment by MotMomet** [comment]
> Yeas, DSF is the best. OreCrunch has now ported to 1.20.4, so here’s hoping that between both these great devs we see a 1.20.1 port (I’ve already asked OreCrunch, they’ve confirmed that they’ll look into doing it in the future provided it isn’t too much work and they get the time). Just thought I’d update anyone here too

hey man if u find the version 1.20.1 in anywhere can u let me know please? lol

**Comment by Rebel459** [comment]
@MotMomet Yeah, sure, will post a link here if it ever gets ported to 1.20.1

**Comment by MotMomet** [comment]
> @MotMomet Yeah, sure, will post a link here if it ever gets ported to 1.20.1

thanks man appreciate it!

**Comment by SnowPowerCore** [comment]
Feel free to visit and download the 1.20.4 version from here: https://modrinth.com/mod/dynamicsurroundingsfabric/versions?g=1.20.4&c=release
Works good.
